pinot-core/src/main/java/com/linkedin/pinot/core/realtime/converter/stats/RealtimeColumnStatistics.java

```diff
@@ -26,15 +31,11 @@
 import com.linkedin.pinot.core.realtime.impl.datasource.RealtimeColumnDataSource;
 import com.linkedin.pinot.core.realtime.impl.dictionary.BaseOnHeapMutableDictionary;
 import com.linkedin.pinot.core.segment.creator.ColumnStatistics;
-import java.util.Arrays;
-import java.util.List;
-import org.apache.commons.lang.math.IntRange;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 
 /**
  * Column statistics for a column coming from an in-memory realtime segment.
+ * TODO Fix this class after we re-factor segment generation code to get some of the stats later on while going through the values
  */
 public class RealtimeColumnStatistics implements ColumnStatistics {
```

Comment:
I will create an issue. This will have to be in the ColumnMetadata class where we should throw exception if the metadata we have is not correct.